bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/Logger.java

```diff
@@ -20,68 +20,140 @@
 
 /**
  * Wrapper around commons Log to conditionally format messages if a specified log level is enabled.
+ *
+ * @author sduskis
+ * @version $Id: $Id
  */
 public class Logger {
   protected final Log log;
 
+  /**
+   * <p>Constructor for Logger.</p>
+   *
+   * @param logClass a {@link java.lang.Class} object.
+   */
   public Logger(Class<?> logClass) {
     this.log = LogFactory.getLog(logClass);
   }
 
+  /**
+   * <p>trace.</p>
+   *
+   * @param message a {@link java.lang.String} object.
+   * @param args a {@link java.lang.Object} object.
```

Comment:
The tool doesn't mention that this is a varag.  Can you please update the various method with varags to clarify?